web/src/components/apps/Dashboard.jsx

```diff
@@ -257,6 +290,7 @@ class Dashboard extends Component {
 
             if (this.props.updateCallback) {
               this.props.updateCallback();
+              this.state.fetchAppDownstreamJob.start(this.fetchAppDownstream, 2000);
```

Comment:
this shouldn't depend on the `updateCallback` function, and should be outside of this if statement